tests/test-mattermost.ts

```diff
@@ -56,8 +63,40 @@ describe('invite users to mattermost', () => {
     .get((uri) => uri.includes('authors.json'))
     .reply(200, testUsers)
     .persist();
-    const inviteUsersToTeamByEmail = mattermostScheduler.inviteUsersToTeamByEmail
-    const result = await inviteUsersToTeamByEmail([...mattermostUsers]);
+    const { inviteUsersToTeamByEmail } = mattermostScheduler;
+    const result = await inviteUsersToTeamByEmail();
     result.length.should.be.equal(2);
   });
+
+  it('create users to team by emails', async () => {
+    nock(/.*ovh.com/)
+    .get(/^.*email\/domain\/.*\/account/)
+    .reply(200, testUsers.map((user) => user.id));
+
+    nock(/.*mattermost.incubateur.net/)
```

Comment:
J'ai un warning sécurité sur cette regex à cause du point "`.`" qui précède le wildcard. 

> This regular expression has an unescaped '.' before 'incubateur.net', so it might match more hosts than expected. 

C'est dans les tests donc je ne pense pas que ça sot grave, mais je me demande si l'expression fait bien ce qu'on veut.

- `/.*mattermost.incubateur.net/` match par exemple _https://false-mattermost.incubateur.net_, tandis que 
- `/^.*\/\/mattermost.incubateur.net` match _http://mattermost.incubateur.net_ et _https://false-mattermost.incubateur.net_ (mais pas _https://false-mattermost.incubateur.net_).